doc/modules/model_evaluation.rst

```diff
@@ -1551,6 +1552,32 @@ Here is a small example of usage of the :func:`mean_absolute_error` function::
   ... # doctest: +ELLIPSIS
   0.849...
 
+.. _mean_absolute_percentage_error:
+
+Mean absolute error
+-------------------
+
+The :func:`mean_absolute_percentage_error` function computes `mean absolute
+percentage error <https://en.wikipedia.org/wiki/Mean_absolute_percentage_error>`_, a risk
+metric corresponding to the expected value of the absolute percentage error loss or
+:math:`l1`-norm of percentage loss.
+
+If :math:`\hat{y}_i` is the predicted value of the :math:`i`-th sample,
+and :math:`y_i` is the corresponding true value, then the mean absolute percentage error
+(MAPE) estimated over :math:`n_{\text{samples}}` is defined as
+
+.. math::
+
+  \text{MAPE}(y, \hat{y}) = \frac{100}{n_{\text{samples}}} \sum_{i=0}^{n_{\text{samples}}-1} \left| \frac{y_i - \hat{y}_i}{y_i} \right|.
+
+Here is a small example of usage of the :func:`mean_absolute_percentage_error` function::
+
+  >>> from sklearn.metrics import mean_absolute_percentage_error
+  >>> y_true = [3, -0.5, 2, 7]
+  >>> y_pred = [2.5, 0.0, 2, 8]
+  >>> mean_absolute_percentage_error(y_true, y_pred)
```

Comment:
maybe add an example of it not being shift-invariant, i.e. add 10 to y_true and y_pred and show that the error is much smaller and add a sentence to explain.

Reply:
done